package.json

```diff
@@ -236,7 +236,8 @@
     "worker-loader": "^3.0.8"
   },
   "resolutions": {
-    "**/mini-store": "^1.1.0"
+    "**/mini-store": "^1.1.0",
+    "**/redux": "3.7.2"
```

Comment:
this is not strictly necessary anymore (was a byproduct of trying to get the library [use-redux-effect](https://github.com/aspect-apps/use-redux-effect) to work). however, I noticed that currently multiple versions of redux are referenced in yarn.lock which doesn't seem ideal (not sure whether they were included in the webpack bundle). so, I thought that pinning the version wouldn't hurt.